scripts/avocado_featurize_submit

```diff
@@ -98,11 +122,21 @@ if __name__ == "__main__":
 
         job_path = '{jobs_directory}/{job_name}.sh'.format(**job_args)
 
-        job_template = sge_template.format(**job_args)
+        if args['schedular'] is None:
```

Comment:
I think that it makes more sense to check if args['schedular'] == 'sge' here explicitly, since it isn't obvious what None means.

Reply:
Resolved with [`3da82c2`](https://github.com/kboone/avocado/pull/6/commits/3da82c2c70431b18ba99bd19dd2d7c8ffdf346cf)